Gems/ROS2/README.md

```diff
@@ -2,34 +2,7 @@
 
 This Gem enables users to develop robotic simulations through ROS2 tools and components.
 
-## Requirements
-
-* Ubuntu 20.04 or 22.04. Other Ubuntu versions and Linux distros could also work as long as they can support ROS 2.
-* [O3DE](https://www.o3de.org/) development branch. The Gem does not work with a release version of O3DE yet.
-* Modern version of ROS 2. This instruction assumes that the `desktop` version is installed. Otherwise some packages might be missing. We support and tested with:
-  * [ROS 2 Galactic](https://docs.ros.org/en/galactic/Installation.html) with Ubuntu 20.04.
-  * [ROS 2 Humble](https://docs.ros.org/en/humble/Installation.html) with Ubuntu 22.04.
-
-#### Additional ros packages required
-
-Once the desired version of ROS 2 is installed, you will need to install additional ROS 2 packages. 
-Before running the following commands, make sure that the ROS 2 environment is setup. Replace \<distro\> with the ROS 2 distribution name (galactic, humble, ..) below and run the following command:
-```
-source /opt/ros/<distro>/setup.bash
-```
-
-* gazebo_msgs: `sudo apt install ros-${ROS_DISTRO}-gazebo-msgs`
-* Ackermann messages: `sudo apt install ros-${ROS_DISTRO}-ackermann-msgs`
-* Control toolbox `sudo apt install ros-${ROS_DISTRO}-control-toolbox`
-* Xacro (XML Macros) `sudo apt install ros-${ROS_DISTRO}-xacro`
-
-If a `desktop` installation of ROS 2 distro was selected, everything else should be there.
-
-Use this helpful command to install:
-
-```
-sudo apt install ros-${ROS_DISTRO}-ackermann-msgs ros-${ROS_DISTRO}-control-toolbox ros-${ROS_DISTRO}-nav-msgs ros-${ROS_DISTRO}-gazebo-msgs
-```
+Documentation about configuring and using this gem can be found [here](https://www.o3de.org/docs/user-guide/interactivity/robotics/).
```

Comment:
Please rephrase to an active form, following the guidelines. This needs to clearly tell the user in an active manner what to do and why.